src/Components/Aspire.Microsoft.EntityFrameworkCore.Cosmos/README.md

```diff
@@ -57,6 +65,8 @@ And then the connection string will be retrieved from the `ConnectionStrings` co
 }
 ```
 
+The `EnrichCosmosDbContext` won't make use of the `ConnectionStrings`, `Region` and `AccountEndpoint` configuration section since it expects a `DbContext` to be registered at the point it is called.
+
 See the [ConnectionString documentation](https://learn.microsoft.com/azure/cosmos-db/nosql/how-to-dotnet-get-started#connect-with-a-connection-string) for more information.
 
 ### Use configuration providers
```

Comment:
The example below still references `"DbContextPooling"`

Reply:
Thanks - https://github.com/dotnet/aspire/pull/2381